xmodule/x_module.py

```diff
@@ -653,12 +651,13 @@ def bind_for_student(self, user_id, wrappers=None):
 
         if wrappers is None:
             wrappers = []
```

Comment:
Nit: we can remove this part because we do the `if wrappers` check below.

Side note: I'm wondering if there are any use cases for not specifying the wrappers at all (outside of tests). I haven't found any.